src/tools/rados/rados.cc

```diff
@@ -82,7 +82,7 @@ void usage(ostream& out)
 "\n"
 "OBJECT COMMANDS\n"
 "   get <obj-name> [outfile]         fetch object\n"
-"   put <obj-name> [infile]          write object\n"
+"   put <obj-name> [infile]  [--obj-offset offset]  write object at offset\n"
```

Comment:
could you use "--offset" instead? as "--obj-offset" is not a global option. i guess one won't be take it as the offset of something else. what do you think?